consensus/istanbul/backend/api.go

```diff
@@ -249,16 +282,15 @@ func (api *APIExtension) getConsensusInfo(block *types.Block) (ConsensusInfo, er
 	// get origin proposer at 0 round.
 	originProposer := common.Address{}
 	lastProposer := common.Address{}
-	// TODO-Klaytn add the logic to get the last proposer for other policies. Weighted Random doesn't need it.
-	if istanbul.ProposerPolicy(snap.Policy) == istanbul.WeightedRandom {
-		newValSet := snap.ValSet.Copy()
-		newValSet.CalcProposer(lastProposer, 0)
-		originProposer = newValSet.GetProposer().Address()
-	} else {
-		logger.Warn("getConsensusInfo does not support to get origin proposal on", "proposerPolicy", snap.Policy)
+	if blockNumber > 0 {
```

Comment:
Whether `blockNumber` == 0 is already checked at the beginning of the method. And, the type of `blockNumber` is uint64. Therefore, the checking logic is not necessary.

Reply:
Good point. I will reflect.